### ISSUE TITLE
poe Blocking Responses to Context-Formatted Messages

### ISSUE THREAD

**Issue by fkx816** [ading2210/poe-api#201]
It appears that POE (assuming this is an application or system name) is blocking responses to context-formatted messages. Even when context-formatted content is sent via the web interface, no response is received. Attempts to send context-formatted content result in a "response timed out" message due to the absence of a reply. See the attached screenshots for specific tests. This issue was discovered by @LanQian in #200. 
Therefore, it may be necessary to advise users not to use these formats, or to test out a context format that can be used.
![Quicker_20230726_025508](https://github.com/ading2210/poe-api/assets/69662155/98dc7567-034f-4b2a-bf68-a1263c5d70c7)

**Comment by fkx816** [comment]
Through my testing, the key is that the end of the message cannot be a colon.
 (However, for some reason, normal sentences ending with a colon are not blocked at times.)

**Comment by isJN** [comment]
I don't think so. The normal conversation is sent normally when using the poe-api, but the error message is that the ws link is closed and the page is viewed as poe without an answer

**Comment by isJN** [comment]
> I don't think so. The normal conversation is sent normally when using the poe-api, but the error message is that the ws link is closed and the page is viewed as poe without an answer

like this
![image](https://github.com/ading2210/poe-api/assets/58289415/80742156-318e-4173-8ace-329e9fa0bc1b)

**Comment by fkx816** [comment]
> I don't think so. The normal conversation is sent normally when using the poe-api, but the error message is that the ws link is closed and the page is viewed as poe without an answer

You can try sending “
User:hello
Assistant:
”from the web to test whether the message is successfully received.
When I said above that the colon is key, I was referring to its occurrence in the context format. The problem you mentioned may not necessarily be due to the context format.

**Comment by LanQian528** [comment]
现在似乎出现了新的阻止方式，同样的消息使用poe-api后claude-2-100k可以回复，而gpt4则不行。然而在网页中使用二者都是可以快速响应的。

**Comment by wingeva1986** [comment]
> 现在似乎出现了新的阻止方式，同样的消息使用poe-api后claude-2-100k可以回复，而gpt4则不行。然而在网页中使用二者都是可以快速响应的。

+1
